### PR TITLE
pgmodeler: init at 0.9.1

### PR DESCRIPTION
**Pull request by esclear**
###### Motivation for this change
[pgModeler](https://pgmodeler.io/) (PostgreSQL Database Modeler) is an open source database modeling tool designed for PostgreSQL.
It is not packaged for NixOS yet, so I did that.

I'm a bit unsure about whether the complete postgresql package is needed as a dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
  (No dependencies)
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
  (+ 563.73 MB)
- [x] Ensured that relevant documentation is up to date
  (No relevant documentation)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers
That'd be me :)

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2045,6 +2045,12 @@
     github = "ertes";
     name = "Ertugrul Söylemez";
   };
+  esclear = {
+    email = "esclear@users.noreply.github.com";
+    github = "esclear";
+    githubId = 7432848;
+    name = "Daniel Albert";
+  };
```
Comment (Lassulus) [inline maintainers/maintainer-list.nix]: can you make a separate commit out this?

Comment (esclear) [reply maintainers/maintainer-list.nix]: Done.

**Comment by Lassulus** [comment]
can you change the commit message to `maintainers: Add @esclear` ?

**Comment by esclear** [comment]
Sure thing.

**Review by Lassulus — APPROVED** [review]
tested with nix-review, opened a good looking graphical frontend where I was overwhelmed by the options I have to design tables and stuff
